src/js/osweb/items/keyboard_response.js

```diff
@@ -1,66 +1,96 @@
-import GenericResponse from './generic_response.js'

```

Comment:
I suppose that your autolinter is responsible for changing all the lines, but this way I cannot see what has changed. And the same goes for `sketchpad.js`. Could you change this in one way or another so that the actual changes are visible?

Reply:
More likely seems that the line endings have changed, otherwise the import statements on the first lines should have remained untouched. Git for Windows messed up the CRLF/LF endings quite frequently until I finally configured it correctly not to do so. I can see if I can reconstruct the correct diffs, but this probably won't be easy and I have to do some searching around how to change items within the commit history.
From the top of my head: the only thing that changed in keyboard_response, mouse_response and sketchpad is the addition of the `* coroutine()` function.

Reply:
Regarding the linting. I am a fierce proponent of everyone adhering to the same code styles and everyone thoughtlessly obeying the linter's wishes (no individual freedom here :) ). To prevent the linter changes obscuring diffs, I propose to let it automatically lint and correct all code once this PR is approved, and then I will send this corrected code as a separate PR. That way, the linter should not have nothing left to change for future commits.

Reply:
> , I propose to let it automatically lint and correct all code once this PR is approved, and then I will send this corrected code as a separate PR. That way, the linter should not have nothing left to change for future commits.

Reply:
I found the reformatting to have happened in commit fcd2e69c4c07cf747c86dbf45eebcd8f98357272, but it's hard to revert it, because files that were affected by this commit have changed after this too. I don't know whether you know a trick to only revert this commit, or that you want to leave it like this. It seems to be a bit wasteful to spend a lot on time on this now. Let's just deal with linter changes in the future as we discussed above?

Reply:
You don't need to revert it. You could, for example, fix the files, and the squash the commit so that only the changes in the end result are visible.

Reply:
I see, but how do I get back to the old situation? Just go back to the point before that commit and copy-paste the contents of the files of that point into the latest version? I don't see how I can undo these format changes, but I'll see if I can find some instructions somewhere.

Reply:
Let me give it a shot.

Reply:
I have committed the 'unchanges' in 7823efc but squashing it with `git rebase --interactive ee19a5fbb6d2fdca0ced7da04480a04fb5b28bf0` (which is when these changes happened), gives me a ton of conflicts. I don't think that should happen, so I must still be doing something incorrectly :sweat:

Reply:
And apologies for the slightly aggressive commit message, but this has taken up quite some time already ;)

Reply:
It's fixed now though, right?

Reply:
I don't know! I cannot see the overview of all changes as I get in your review any longer. I cannot find this option anywhere in the interface?

Reply:
Ah wait... Files changed :sweat_smile: 
Appears to be fixed!